bpf/lib/bpf_cred.h

```diff
@@ -69,4 +69,37 @@ struct msg_cred_minimal {
 	__u32 pad;
 } __attribute__((packed));
 
+/* Execution and cred related flags shared with userspace */
+#define EXEC_SETUID    0x01 /* This is a set-user-id execution */
+#define EXEC_SETGID    0x02 /* This is a set-group-id execution */
+#define EXEC_FS_CAPS   0x04 /* This binary execution gained new capabilities through file capabilities execution */
+#define EXEC_FS_SETUID 0x08 /* This binary execution gained new privileges through setuid to root execution */
```

Comment:
nit: no need to change anything, just something to be aware of.

I find the naming confusing. Here's my understanding based of the comments:
* We use `EXEC_SETUID` for all setuid binaries
* We use `EXEC_FS_SETUID` for execution of setuid binaries that lead to privilege elevation

Does the `FS` stand for filesystem? The reason I find it confusing is that, if my understanding is correct, `EXEC_SETUID` is also related to filesystem properties.

Reply:
yes makes sense, those were internal vars but I renamed to be EXEC_SETUID_ROOT